.eslintrc.json

```diff
@@ -13,7 +13,11 @@
     "project": "./lint.tsconfig.json"
   },
   "rules": {
-    "import/order": "warn"
+    "import/order": "warn",
+    "@typescript-eslint/no-confusing-void-expression": [
+      "error",
+      { "ignoreArrowShorthand": true }
```

Comment:
IMO we want to continue being able to do things like `.map(obj => obj.val)` and in order to do so we have to explicitly set `ignoreArrowShorthand` to true.  More details can be found here: https://typescript-eslint.io/rules/no-confusing-void-expression/#ignorearrowshorthand

Reply:
Huh, ok. Yeh this seems like a good one? Probs should add this to our template.